src/browserview/BrowserComponent.java

```diff
@@ -418,6 +427,18 @@ private void bringToTop(){
 			user32.SetForegroundWindow(browserWindowHandle);
 		} 
 	}
+
+	public void bringToFront() {
+		if (PlatformSpecific.isOnMac()) {
+			try {
+				scriptEngine.eval("tell application \"System Events\" to repeat with p in (every application process whose name contains \"Chrome\" and name does not contain \"Helper\")\nif (title of window of p as string) contains \"" + driver.getTitle() + "\" then tell process p to perform action \"AXRaise\" of window 1 of p\nend repeat");
+			} catch (ScriptException e) {
+				logger.info("Bring browser window to front script exception! " + e.getLocalizedMessage());
+			} catch (Exception e) {
+				logger.info("Bring browser window to front exception! " + e.getLocalizedMessage());
+			}
```

Comment:
it's more for debugging purposes (because i think applescript is notoriously flaky), should i remove them then?